sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -93,7 +93,7 @@
 
 > These changes do not impact the API of stable versions such as 11.3.0.
 > Only code written against a beta version such as 11.4.0b6 may be affected.
-- Renamed `vector_search_configuration` to `vector_search_profile` in `SearchField`.
+- Renamed `vector_search_configuration` to `vector_search_profile_name` in `SearchField`.
```

Comment:
I think the case is we renamed `vector_search_profile` to `vector_search_profile_name` in the GA release?

Reply:
Yes. Seems like we missed it in the GA release breaking change section.

Reply:
Okay added to the GA breaking changes section.